datanode/dehistory/snapshot/service_create_snapshot.go

```diff
@@ -61,30 +65,44 @@ func (b *Service) CreateSnapshot(ctx context.Context, chainID string, fromHeight
 	conn, err := pgxpool.Connect(ctx, b.connConfig.GetConnectionString())
 	if err != nil {
 		runAllInReverseOrder(cleanUp)
-		return CreateSnapshotResult{}, fmt.Errorf("unable to connect to database: %w", err)
+		return MetaData{}, fmt.Errorf("unable to connect to database: %w", err)
 	}
 	cleanUp = append(cleanUp, func() { conn.Close() })
 
 	copyDataTx, err := conn.Begin(ctx)
 	if err != nil {
 		runAllInReverseOrder(cleanUp)
-		return CreateSnapshotResult{}, fmt.Errorf("failed to begin copy table data transaction: %w", err)
+		return MetaData{}, fmt.Errorf("failed to begin copy table data transaction: %w", err)
 	}
 	// Rolling back a committed transaction does nothing
 	cleanUp = append(cleanUp, func() { _ = copyDataTx.Rollback(ctx) })
 
 	if _, err = copyDataTx.Exec(ctx, "SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
 		runAllInReverseOrder(cleanUp)
-		return CreateSnapshotResult{}, fmt.Errorf("failed to set transaction isolation level to serilizable:%w", err)
+		return MetaData{}, fmt.Errorf("failed to set transaction isolation level to serilizable:%w", err)
```

Comment:
Can we just make sure we have a space before the `%w` to make the errors more readable? i.e `: %w`?